### PR TITLE
Disable CSIMigration* feature gates for kube-proxy

### PR DESCRIPTION
**Pull request by Cynerva**
See https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/193 for context.

### CODE REVIEW

File: lib/charms/layer/kubernetes_common.py [hunk 1]
```diff
@@ -647,8 +647,19 @@ def configure_kube_proxy(
     if host.is_container():
         kube_proxy_opts["conntrack-max-per-core"] = "0"
 
+    features_gates = []
```
Comment (addyess) [inline lib/charms/layer/kubernetes_common.py]: oops, misspelling
```suggestion
    feature_gates = []
```

**Review by addyess — CHANGES_REQUESTED** [review]
spelling oopsie

**Comment by Cynerva** [comment]
Fixes https://bugs.launchpad.net/bugs/1954807

File: lib/charms/layer/kubernetes_common.py (hunk 1 above)
Comment (Cynerva) [reply lib/charms/layer/kubernetes_common.py]: Oops. Thanks.

**Review by addyess — APPROVED** [review]

**Pull request merged**